tasks/tslint.js

```diff
@@ -1,100 +1,102 @@
-        var done = this.async();
-        var failed = 0;
-
-        // Iterate over all specified file groups, async for 'streaming' output on large projects
-        grunt.util.async.reduce(this.filesSrc, true, function (success, filepath, callback) {
-            if (!grunt.file.exists(filepath)) {
-                grunt.log.warn('Source file "' + filepath + '" not found.');
-            } else {
-                var configuration = specifiedConfiguration;
-                if (configuration == null || typeof configuration === "string") {
-                    configuration = Linter.findConfiguration(configuration, filepath);
-                }
-                options.configuration = configuration;
-
-                var contents = grunt.file.read(filepath);
-                var linter = new Linter(filepath, contents, options);
-                var result = linter.lint();
-
-                if (result.failureCount > 0) {
-                    var outputString = "";
-                    var outputFile = options.outputFile;
-                    var appendToOutput = options.appendToOutput;
-
-                    failed += result.failureCount;
-
-                    if (outputFile != null && grunt.file.exists(outputFile)) {
-                        if (appendToOutput) {
-                            outputString = grunt.file.read(outputFile);
-                        } else {
-                            grunt.file.delete(outputFile);
-                        }
-                    }
-                    result.output.split("\n").forEach(function (line) {
-                        if (line !== "") {
-                            if (outputFile != null) {
-                                outputString += line + "\n";
-                            } else {
-                                grunt.log.error(line);
-                            }
-                        }
-                    });
-                    if (outputFile != null) {
-                        grunt.file.write(outputFile, outputString);
-                    }
-                    success = false;
-                }
-            }
-
-            // Using setTimout as process.nextTick() doesn't flush
-            setTimeout(function () {
-                callback(null, success);
-            }, 1);
-
-        }, function (err, success) {
-            if (err) {
-                done(err);
-            } else if (success) {
-                var okMessage = this.filesSrc.length + " " + grunt.util.pluralize(this.filesSrc.length, "file/files") + " lint free.";
-                grunt.log.ok(okMessage);
-                done();
-            } else {
-                var errorMessage = failed + " " + grunt.util.pluralize(failed, "error/errors") + " in " +
-                    this.filesSrc.length + " " + grunt.util.pluralize(this.filesSrc.length, "file/files");
-                grunt.log.error(errorMessage);
-                done(false);
-            }
-        }.bind(this));
-    });
-};
+/*

```

Comment:
This file also shows up with every line being different. If you can figure out how to fix that it would be great - would make it easier to review